pages/events/fall-fest.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <article class="fall-fest-page">
+    <LayoutLeadSpaceWithCard
+      :cta="headerData.cta"
+      :card-title="headerData.cardSectionHeading"
+    >
+      <template #title>
+        {{ headerData.titleLine1 }}
+      </template>
+      <template #description>
+        <p>
+          The Qiskit Fall Fest is a collection of quantum computing events on
```

Comment:
You can add this description to the content file under `header`, with the rest of the content

Reply:
Addressed [here](https://github.com/Qiskit/qiskit.org/pull/3525/commits/f097177d9aa8d68f1b2a9f1a422b786226ef4f0f)